template.go

```diff
@@ -69,14 +70,9 @@ func RunExpression(environment map[string]any, template Template) (any, error) {
 		return "", err
 	}
 
-	data, err := Serialize(environment)
-	if err != nil {
-		return "", err
-	}
```

Comment:
We don't need to serialize anymore for the expressions? I think that would have provided a consistent variable addressing in expression and template?